.github/workflows/reusable_deploy_docs.yml

```diff
@@ -220,3 +214,4 @@ jobs:
           vercel_team_name: ${{ vars.VERCEL_TEAM_NAME }}
           vercel_project_name: ${{ vars.VERCEL_PROJECT_NAME }}
           release_commit: ${{ inputs.RELEASE_COMMIT }}
+
```

Comment:
:eyes:

Reply:
My editor adds these https://github.com/rerun-io/rerun/issues/2741 should happen at some point